scripts/qa_crowbarsetup.sh

```diff
@@ -1483,29 +1483,83 @@ EOF
     return 0
 }
 
-# run the crowbar install script
-# and do some sanity checks on the result
-function do_installcrowbar()
+function crowbar_install_status()
+{
+    curl -s $crowbar_api/installer/status.json | python -mjson.tool
+}
+
+function do_installcrowbar_cloud6plus()
+{
+    service crowbar status || service crowbar stop
+    service crowbar start
+
+    wait_for 30 10 "curl -s $crowbar_api/installer | grep -q '/installer/install'" "crowbar installer to be available"
+
+    if crowbar_install_status | grep -q '"success": *true' ; then
+        echo "Crowbar is already installed. The current crowbar install status is:"
+        crowbar_install_status
+        return 0
+    fi
+
+    # call api to start asyncronous install job
+    curl -s -X POST $crowbar_api/installer/install || complain 39 "crowbar is not running"
+
+    wait_for 60 10 "crowbar_install_status | grep -q '\"success\": *true'" "crowbar to get installed" "tail -n 500 $crowbar_install_log"
+}
+
+
+function do_installcrowbar_legacy()
 {
     local instparams="$1 --verbose"
-    pre_hook $FUNCNAME
     local instcmd
     if [ -e /tmp/install-chef-suse.sh ]; then
         instcmd="/tmp/install-chef-suse.sh $instparams"
     else
         instcmd="/opt/dell/bin/install-chef-suse.sh $instparams"
     fi
-    do_set_repos_skip_checks
 
     cd /root # we expect the screenlog.0 file here
     echo "Command to install chef: $instcmd"
     intercept "install-chef-suse.sh"
 
     rm -f /tmp/chef-ready
-    rpm -Va crowbar\*
```

Comment:
why is that line removed? you're doing it before and after in the new installer as well..